api/cluster/cluster.proto

```diff
@@ -3,9 +3,6 @@ syntax = "proto3";
 package cluster;
 
 option go_package = "github.com/talos-systems/talos/pkg/machinery/api/cluster";
-option java_multiple_files = true;
-option java_outer_classname = "ClusterApi";
-option java_package = "com.cluster.api";
```

Comment:
Those options were not present everywhere (health.proto, for example), and cluster.com costs $150k at least: https://domaininvesting.com/cluster-com-acquired-by-namecheap-for-150k/